ui/src/app/views/signed-in/component.html

```diff
@@ -1,4 +1,4 @@
-<div class="page-container">
+<div *ngIf="!inNotebookFrame" class="page-container">
```

Comment:
Could you add a high-level description of the approach you are using to the PR description? I think I see it now, but I wasn't initially understanding why you'd need to do this - are you loading our application inside the iframe initially?

An alternative would be to take the existing notebook redirect page, and instead of redirecting at the end show the iframe. Do we plan to continue supporting the full page redirect -> Jupyter after we have iframe support?

Reply:
Maybe we should chat, as I'm a little confused by your last question. I could implement differently as far as the notebook redirect page, but since the notebookframecomponent is still in our app, it would render with the our header, which is what I'm attempting to avoid here.